lib/mods/replace-methods.js

```diff
@@ -54,12 +55,14 @@ function isNamespaceMethod(node, baseTypeName) {
  * @param {Object} imports - the current list of imports
  */
 function replaceGlobalMethodCalls(node, imports) {
-  if (nodeTypes.isPropertyAccessExpression(node) &&
-      nodeTypes.isCallExpression(node.getParent()) &&
-      isGlobal(node.getExpression())) {
-    const method = node.getSymbol().getEscapedName();
+  if (node.getExpression &&
+      nodeTypes.isPropertyAccessExpression(node.getExpression()) &&
+      nodeTypes.isCallExpression(node) &&
+      isGlobal(node.getExpression().getExpression())) {
```

Comment:
I think if you move `nodeTypes.isCallExpression()` first, then you will not need to test `node.getExpression`.

Also, it would be good to factor this out into `isGlobalMethodCall` like you did for get/set.